assets/js/modules/adsense/components/settings/SettingsForm.js

```diff
@@ -125,9 +128,12 @@ export default function SettingsForm() {
 
 			<AutoAdExclusionSwitches />
 
-			<AdBlockingRecoveryCTA />
-
-			<AdBlockingRecoveryToggle />
+			{ adBlockerDetectionEnabled && (
+				<Fragment>
+					<AdBlockingRecoveryCTA />
```

Comment:
This component is also rendered in the `SettingsView` which is still being rendered unconditionally.

https://github.com/google/site-kit-wp/blob/94ecff3e5ac9587fed96afde2989f3849f512df8/assets/js/modules/adsense/components/settings/SettingsView.js#L225-L228 

Apologies, I missed this in the IB too.